recipes/citrix-workspace/webview.js

```diff
@@ -0,0 +1,8 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
```

Comment:
This should be removed unless you are planning on adding a `.css` file, since this is not used anywhere in this script at the moment.